packages/test/src/test-testenvironment.ts

```diff
@@ -20,153 +21,155 @@ interface Context {
 
 const test = anyTest as TestFn<Context>;
 
-test.before(async (t) => {
-  t.context = {
-    testEnv: await TestWorkflowEnvironment.createTimeSkipping(),
-    bundle: await bundleWorkflowCode({
-      workflowsPath: require.resolve('./workflows/testenv-test-workflows'),
-      workflowInterceptorModules,
-    }),
-  };
-});
-
-test.after.always(async (t) => {
-  await t.context.testEnv?.teardown();
-});
-
-test.serial('TestEnvironment sets up test server and is able to run a Workflow with time skipping', async (t) => {
-  const { client, nativeConnection } = t.context.testEnv;
-  const worker = await Worker.create({
-    connection: nativeConnection,
-    taskQueue: 'test',
-    workflowBundle: t.context.bundle,
-  });
-  await worker.runUntil(
-    client.workflow.execute(sleep, {
-      workflowId: uuid4(),
-      taskQueue: 'test',
-      args: [1_000_000],
-    })
-  );
-  t.pass();
-});
-
-test.serial('TestEnvironment can toggle between normal and skipped time', async (t) => {
-  const { client, nativeConnection } = t.context.testEnv;
-
-  const worker = await Worker.create({
-    connection: nativeConnection,
-    taskQueue: 'test',
-    workflowBundle: t.context.bundle,
+if (RUN_TIME_SKIPPING_TESTS) {
```

Comment:
Can you use the same trick you used above?

```ts
const testTimeSkipping = RUN_TIME_SKIPPING_TESTS ? test : test.skip
```

You could possibly import that from helpers.